security/pkg/pki/ca/ca.go

```diff
@@ -68,8 +68,13 @@ const (
 
 // SigningCAFileBundle locations of the files used for the signing CA
 type SigningCAFileBundle struct {
-	RootCertFile    string
-	CertChainFiles  []string
+	// RootCertFile is the detected file holding multiple root CAs
+	RootCertFile string
+	// CertChainFile is the detected file holding the intermeidary certificates (old style)
+	// and the tls.crt holding both the leaf cert and intermdediaries.
```

Comment:
```suggestion
	// CertChainFile is the detected file holding the intermediary certificates (old style)
	// and the tls.crt holding both the leaf cert and intermediaries.
```

Could you clarify this comment (`and` vs `or`)? Thanks!